src/version.cc

```diff
@@ -23,7 +23,7 @@
 
 #define NG_TF_MAJOR_VERSION 0
 #define NG_TF_MINOR_VERSION 14
-#define NG_TF_PATCH_VERSION 0
+#define NG_TF_PATCH_VERSION 1
```

Comment:
This will probably become 0.14.2, so this will probably be `2` here. but not sure, maybe lets check with @avijit-nervana

This comment applies to all places where we have 0.14.1

Reply:
No - this is targeted for master - so don't change this. The master will be merged to r0.15.